app/scripts/modules/datepicker.js

```diff
@@ -92,8 +94,8 @@ angular.module('modified.datepicker', ['strap.position'])
         showToday: true,
         startingDay: 0,
         yearRange: 20,
-        minDate: null,
-        maxDate: null
+        minDate: minDate,
```

Comment:
make this `new Date(new Date().getFullYear(), 0, 1)`

maxDate to ` new Date()`